docs/src/main/paradox/scala/http/routing-dsl/source-streaming-support.md

```diff
@@ -115,3 +115,14 @@ instances, which can be provided using any JSON marshalling library (such as Cir
 
 When implementing a custom support trait, one should simply extend the `EntityStreamingSupport` abstract class,
 and implement all of it's methods. It's best to use the existing implementations as a guideline.
+
+## Supporting custom content types
+
+In order to marshal into custom content types both a `Marshaller` that can handle that content type, 
+**as well as an `EntityStreamingSupport` of matching content type** is required.
```

Comment:
Certainly. It just triggered one of the points that got me to start on the streaming approach on the first place. I agree it's unrelated to this PR.